internal/hcl/modules/fetch.go

```diff
@@ -66,6 +65,10 @@ func NewPackageFetcher(remoteCache RemoteCache, logger zerolog.Logger, opts ...P
 		opt(p)
 	}
 
+	if p.timeout > 0 {
+		getters["git"].(*CustomGitGetter).GitGetter.Timeout = p.timeout
```

Comment:
Any chance we can change it at the point of init: https://github.com/infracost/infracost/blob/master/internal/hcl/modules/fetch.go#L56

e.g. 

```go
 &CustomGitGetter{getter.GitGetter{Timeout: p.timeout} }
 ```

Reply:
I didn't do that as there's an option that can modify it (`WithTimeout` or something), but lets simplify and remove the option as nothing uses it 👍